cmd/process-agent/main_windows.go

```diff
@@ -116,21 +115,19 @@ func runService(isDebug bool) {
 // main is the main application entry point
 func main() {
 	ignore := ""
-	flag.StringVar(&opts.configPath, "config", defaultConfigPath, "Path to datadog.yaml config")
-	flag.StringVar(&opts.sysProbeConfigPath, "sysprobe-config", defaultSysProbeConfigPath, "Path to system-probe.yaml config")
-	flag.StringVar(&ignore, "ddconfig", "", "[deprecated] Path to dd-agent config")
-	flag.BoolVar(&opts.info, "info", false, "Show info about running process agent and exit")
-	flag.BoolVar(&opts.version, "version", false, "Print the version and exit")
-	flag.StringVar(&opts.check, "check", "", "Run a specific check and print the results. Choose from: process, connections, realtime")
+	rootCmd.PersistentFlags().StringVar(&opts.configPath, "config", defaultConfigPath, "Path to datadog.yaml config")
```

Comment:
I could be wrong, but doesn't it result in having all flags with `--` as opposed to `-`? Could you please confirm?

Reply:
It does, but as far as I could tell, Cobra isn't really compatible with the standard library's flags. I'm honestly not really sure what we should do here, so we should probably discuss this

Reply:
right, you can't change it because it will break existing installations.

Reply:
Looking into it, it looks like I was wrong, and cobra actually does have support for non-posix style argument parsing. I'll spend some time fixing this and let you know when it's done.